content/installation/bootstrapping.md

```diff
@@ -335,24 +335,16 @@ pip install https://github.com/cloudify-cosmo/cloudify-cli/archive/4.1.zip
 {{< /gsHighlight >}}
 
 #### Step 4: Download the YAML Files and DSL Resources 
-The simple-manager-blueprint imports two YAML files and, by default, uploads a number of DSL resources to the Manager. Download all the files to the same base directory.   <br>
-   **YAML files**   <br>
+The simple-manager-blueprint imports two YAML files and, by default, uploads a number of DSL resources to the Manager. 
 
-      * http://www.getcloudify.org/spec/cloudify/4.1/types.yaml
-      * http://cloudify.co/spec/fabric-plugin/1.5/plugin.yaml
-
-   **DSL resources**   <br>
-
-      * http://cloudify.co/spec/openstack-plugin/2.0.1/plugin.yaml
-      * http://cloudify.co/spec/aws-plugin/1.4.10/plugin.yaml
-      * http://www.getcloudify.org/spec/tosca-vcloud-plugin/1.3.1/plugin.yaml
-      * http://cloudify.co/spec/vsphere-plugin/2.3.0/plugin.yaml
-      * http://cloudify.co/spec/diamond-plugin/1.3.5/plugin.yaml
+Download all these files to the same base directory:
 
    {{< gsHighlight  bash  >}}
-   cd ~/cloudify/offline
-   mkdir plugins && cd plugins
-   curl -L -O http://repository.cloudifysource.org/cloudify/wagons/cloudify-fabric-plugin/1.5/cloudify_fabric_plugin-1.5-py27-none-linux_x86_64-centos-Core.wgn
+cd ~/cloudify/offline
+mkdir dsl && cd dsl
```

Comment:
is this correct?

Reply:
Seems correct to me. See section 6 part 2 below.

Reply:
@isaac-s Can you help reviewing this?